source/jormungandr/tests/instant_system_new_default_routing_tests.py

```diff
@@ -217,7 +199,9 @@ def test_basic_ride_sharing(self):
 
         assert rsj_links[1].get('rel') == 'tickets'
         assert rsj_links[1].get('type') == 'ticket'
-        assert rsj_links[1].get('id') == ticket_id
+        assert rsj_links[1].get('id') == ticket['id']
+        assert ticket['links'][0]['id'] == rsj_sections[1]['id']
+        assert rs_journeys[0].get('fare').get('total').get('value') == tickets[0].get('cost').get('value')
 
         assert rsj_sections[2].get('type') == 'crow_fly'
```

Comment:
minor: maybe we can test a bit more check that we have an `id`, and also that `from`/`to` are correct objects, matching the ones around... Can be done in another PR.